### ISSUE TITLE
remove affiliated string from copied link 

### ISSUE THREAD

**Issue by terra921** [garoloup/SendToFreebox#8]
When you copy a link from Zone Telechargement , the link is ended with an affiliated sting "&af<number>" .
This prevents from downloading the link in the Freebox File system .
If we remove this string manually , then the link works .
An enhancement is to remove automatically this affiliated information at the end of link.

Thanks
Manuel

**Comment by Aymkdn** [comment]
Je peux modifier le code source, mais c'est @garoloup qui l'a publié dans le store avec son identifiant. Je ne sais pas s'il est encore actif / lit encore ce qu'il se passe ici ?

Sinon, je publierai ma propre version dans le store.

**Comment by garoloup** [comment]
Je ne suis plus actif sur ce code car plus chez Free.
Si besoin je peux publier les modifications.
Sinon Aymkdn il faut ouvrir un compte développement 
Par ailleurs il y a aussi potentiellement des évolutions à faire pour rendre compatible avec manifest v3 autrement l’appli va disparaître

**Comment by Aymkdn** [comment]
OK Merci.

Oui, j'ai déjà un compte sur les stores car j'ai d'autres extensions.

Du coup, je regarde ça quand j'ai un moment, et pour la mise en place du manifest v3.

**Comment by terra921** [comment]
Bonjour 

en fait il faut rechercher la chaine &af= , l'effacer et tout ce qu'il y a apres .
Merci pour votre réponse et votre aide .
Manu

**Comment by Aymkdn** [comment]
Oui, bien compris @terra921 mais cette extension est ancienne, et pour la republier il va certainement falloir la mettre à jour vers les nouvelles règles imposées par les stores, ce qui va demander pas mal de boulot ! Donc même si ça demande est mineure, cela implique d'autres modifications plus profondes.

**Comment by terra921** [comment]
OK . pas de pression :-) 
Merci
Manu

**Comment by Aymkdn** [comment]
Ça m'aura pris quelques mois, mais j'ai créé une nouvelle extension, compatible avec Manifest v3 (que Chrome demande désormais). Je vais la tester chez moi (en particulier avec Firefox qui en retard par rapport à Chrome sur la norme) ces prochains jours et si c'est bon je la publierai.

@terra921 : sur https://www.zone-telechargement.tools je ne vois pas de Torrent ? Tu peux m'indiquer sur quel site et à quel endroit exactement tu trouves les liens qui ont un "&af=" ? Ça me permettra de faire des tests.

**Comment by Aymkdn** [comment]
J'ai soumis la nouvelle extension dans le store de Firefox et Chrome. J'attends de voir ce que ça va donner.

Pour ton cas @terra921 j'ai mis dans les options de l'application le moyen d'indiquer une expression régulière pour remplacer ce qu'on veut dans les URL. Ça devrait couvrir ton cas et d'autres.

En attendant l'extension est disponible sur https://github.com/Aymkdn/download-with-freebox/
